tests/integration/use_cases/test_tag.py

```diff
@@ -0,0 +1,55 @@
+from kili.adapters.kili_api_gateway import KiliAPIGateway
+from kili.use_cases.tag import TagUseCases
+
+
+def test_when_get_tags_of_organization_then_i_get_tags(kili_api_gateway: KiliAPIGateway):
+    # Given
+    tags = [
+        {"id": "tag1_id", "label": "tag1"},
+        {"id": "tag2_id", "label": "tag2"},
+    ]
+    kili_api_gateway.list_tags_by_org.return_value = tags
+
+    # When
+    fetched_tags = TagUseCases(kili_api_gateway).get_tags_of_organization(fields=["id", "label"])
+
+    # Then
+    assert fetched_tags == tags
+
+
+def test_when_get_tags_of_project_then_i_get_tags(kili_api_gateway: KiliAPIGateway):
+    # Given
+    tags = [
+        {"id": "tag1_id", "label": "tag1"},
+        {"id": "tag2_id", "label": "tag2"},
+    ]
+    kili_api_gateway.list_tags_by_project.return_value = tags
+
+    # When
+    fetched_tags = TagUseCases(kili_api_gateway).get_tags_of_project(
+        project_id="fake_proj_id", fields=["id", "label"]
+    )
+
+    # Then
+    assert fetched_tags == tags
+
+
+def test_when_tagging_project_then_it_tags_the_project(kili_api_gateway: KiliAPIGateway):
```

Comment:
we can add a test to assert that it raises an error when a tag is not found in the orga tags